code/modules/client/client procs.dm

```diff
@@ -537,6 +547,14 @@
 			message_admins("SQL ERROR during log_client_to_db (update). Error : \[[err]\]\n")
 	else
 		//New player!! Need to insert all the stuff
+
+		// Check new peeps for panic bunker
+		if(GLOB.panic_bunker_enabled)
+			message_admins("<span class='adminnotice'>Failed Login: [key] - New account attempting to connect during panic bunker</span>")
```

Comment:
Seems like it will cause a lot of log spam

Reply:
So do many other things, such as attack logs. This isnt that big of an issue

Reply:
Attack logs have levels where we can filter out what is usually just spam. 

Why do admins need to be notified of each individual person? We're already notified that the panic bunker is on.

Reply:
So they can prepare for specific people questioning why they cant get in? This is how TG did it so.........